src/jats/jats-exporter.ts

```diff
@@ -2158,7 +2162,8 @@ export class JATSExporter {
 
   sectionToFootnote = (section: Element, fnType: string) => {
     const footNote = this.document.createElement('fn')
-    footNote.setAttribute('fn-type', fnType)
+    const footnoteType = chooseJatsFnType(fnType)
+    footNote.setAttribute('fn-type', footnoteType)
```

Comment:
This should happen in the serializer.

Reply:
I processed the footnotes here since this section soul purpose is to process the footnotes.

Re- (I think we should add a new moveSectionsToFront similar to moveSectionsToBack ...)
I wanted to, but it seemed like it gonna add more complexity to changes.